### PR TITLE
Render link to a source code in rule doc

### PR DESCRIPTION
**Pull request by mvorisek**
Having a source link to the source code of each rule is useful for developer to see how the rule works and for possible improvements/fixes. Currently, as the rule names are different than the fixer class names, it is quite hard to find the source codes easily.

WIP/TODO

- Please advise how to format
- Also for Rule set doc pages
- Please advise how to compute a relative pathes

### CODE REVIEW

File: src/Documentation/DocumentationGenerator.php [hunk 1]
```diff
@@ -70,6 +70,17 @@ public function getFixersDocumentationIndexFilePath()
         return $this->getFixersDocumentationDirectoryPath().'/index.rst';
     }
 
+    /**
+     * @return string
+     */
+    public function makeRelativePathToDoc(string $absolutePath, int $docDepth)
```
Comment (keradus) [inline src/Documentation/DocumentationGenerator.php:76]: why public ?

Comment (mvorisek) [reply src/Documentation/DocumentationGenerator.php:76]: will fix

File: src/Documentation/DocumentationGenerator.php [hunk 2]
```diff
@@ -174,6 +185,9 @@ public function generateFixerDocumentation(FixerInterface $fixer)
 
         $doc = "{$titleLine}\n{$title}\n{$titleLine}";
 
+        $srcPath = (new \ReflectionClass($fixer))->getFileName();
+        $doc .= "\n\n`src <{$this->makeRelativePathToDoc($srcPath, 2)}>`_";
```
Comment (keradus) [inline src/Documentation/DocumentationGenerator.php:189]: if `src`, them maybe also `tests` ?

Comment (mvorisek) [reply src/Documentation/DocumentationGenerator.php:189]: why not ;-)

what is the best way to get a test path for a given fixer class?

Comment (keradus) [reply src/Documentation/DocumentationGenerator.php:189]: reflection->getFileName is good

Comment (keradus) [reply src/Documentation/DocumentationGenerator.php:189]: my reasoning is that actually from `tests` we get a lot of examples, from `src` file we don't get that much as users

**Comment by mvorisek** [comment]
closing as doc directory structure is the same as the src/tests

**Pull request closed without merging**